src/services/api.tsx

```diff
@@ -39,7 +44,109 @@ export const getPatients = async () => {
         if (name[0].given) patient.firstName = name[0].given.join(", ");
         if (name[0].family) patient.lastName = name[0].family;
       }
+      if (identifier) {
+        patient.identifier = identifier
+          .map((e: any) => {
+            return e.value;
+          })
+          .join(", ");
+      }
+
       return patient;
     }
   );
 };
+
+export const getPatientData = (patientId: string) => {
+  /*
+    getPatientData requests for data from Patient resourcce of id patientId
+    return a Patient object.
+  */
+  const getPatientData = async () => {
+    let response = await client.search({
+      type: "Patient",
+      patient: patientId,
+      query: {}
+    });
+    if (!response.data.entry) return; //patient not found
```

Comment:
does this work ? if `response.data.entry` is an empty array, this condition will return false --> `!![] === true`

Reply:
you can use if (!response.data.entry.length) instead

Reply:
response.data.entry is undefined when there are no results.